spec.bs

```diff
@@ -2192,17 +2286,20 @@ The <dfn http-header><code>X-fledge-bidding-signals-format-version</code></dfn>
 is a [=structured header=] whose value must be an [=structured header/integer=].
 
 <div algorithm>
-To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, and a [=boolean=] |isBiddingSignal|:
+To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, an [=origin=] |scriptOrigin|,
+and a [=boolean=] |isBiddingSignal|:
 
   1. Let |request| be a new [=request=] with the following properties:
     :   [=request/URL=]
     ::  |url|
+    :   [=request/origin=]
+    ::  |scriptOrigin|
     :   [=request/header list=]
     ::  «`Accept`: `application/json`»
     :   [=request/client=]
     ::  `null`
     :   [=request/mode=]
-    ::  "`no-cors`"
+    ::  "`cors`"

```

Comment:
It's in canary/dev experiment right now (might go to beta shortly). My expectation is that the only observable change here when the URL is already same-origin is sec-fetch-mode going no-cors => cors. 
Am I missing something?

I can't seem to find the I2S (I think the spec might be a prereq? this process stuff confuses me)

Reply:
>My expectation is that the only observable change here when the URL is already same-origin is sec-fetch-mode going no-cors => cors.

Just to confirm - if one's production deployed TBS endpoint does not handle CORS (because this was not expected) then its response will not be processed when this feature is enabled, correct?

Reply:
> Just to confirm - if one's production deployed TBS endpoint does not handle CORS (because this was not expected) then its response will _not_ be processed when this feature is enabled, correct?

If you keep using a same-origin URL this should not be a problem. I only expect trouble if your endpoint specifically checks the sec-fetch-mode request header's value. Otherwise things should behave the same as before, as it's a simple, same-origin request.

Reply:
Well, the cross-origin case is exactly the new case added by this overall spec change. Previously it would be rejected early on when checking the auctionConfig or interestGroup

Reply:
OK got it. I guess I didn't realize that. Thanks.